atlas/packages/app/config-overrides.js

```diff
@@ -0,0 +1,46 @@
+const path = require('path')
+const { override, addBabelPreset, addWebpackAlias, disableEsLint } = require('customize-cra')
+const eslintConfig = require('../../.eslintrc.js')
+
+// based on https://github.com/arackaf/customize-cra/issues/175#issuecomment-610023655
+const useEslintConfig = (configRules) => (config) => {
+	const updatedRules = config.module.rules.map((rule) => {
+		// Only target rules that have defined a `useEslintrc` parameter in their options
+		if (rule.use && rule.use.some((use) => use.options && use.options.useEslintrc !== void 0)) {
```

Comment:
Any specific reason for `void 0` instead of `undefined` here?

Reply:
This code is copied over from `customize-cra`, that's the check they use there, I didn't question whys